server/user_config.go

```diff
@@ -115,6 +119,29 @@ type UserConfig struct {
 	WebsocketCheckOrigin   bool            `mapstructure:"websocket-check-origin"`
 }
 
+// ToAllowCommandNames parse AllowCommands into a slice of CommandName
+func (u UserConfig) ToAllowCommandNames() ([]command.Name, error) {
+	var allowCommands []command.Name
+	var hasAll bool
+	for _, input := range strings.Split(u.AllowCommands, ",") {
+		if input == "all" {
+			hasAll = true
+			continue
```

Comment:
Shouldnt this be a `break`?

Reply:
`--allow-commands=all,invalid` should become error, so continue is better.

Reply:
Why error? Shouldnt `all` or `*` stomp over any other values?

Reply:
If use `break`, `--allow-commands=all,invalid` become success and `--allow-commands=invalid,all` become error.  To become same behaviour, both cases should be error.

Reply:
Ah ok so then if you stick with continue and all does not override the other values then that should be documented

Reply:
Add more test cases too.

https://github.com/runatlantis/atlantis/pull/2877/commits/74fcd4016de55c10ad36a1503ed8c668734feca2